### DISCUSSION TITLE
Impossible to delete some files

### DISCUSSION THREAD

**Discussion by kammardekamm** [aclap-dev/video-downloadhelper#1248] [Help and Q&A]
### Your question

Hi,

if I use downloadhelper and stop the downloading, then it's impossible to delete the files (see the attached file). They remain in "stop" status.
I tried my best to delete them, but I didn't manage.  
Can you help me please?
![IMPOSSIBLE TO DELETE FILES IN STOP STATUS](https://github.com/user-attachments/assets/43728c24-c212-42eb-8fc5-17a1845161f8)

### Addon & CoApp version and details

version: 9.1.0.43
target: microsoft
channel: stable
lang: it
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"b5feb1eabd9e47e2"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 Edg/128.0.0.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "concurrent_downloads_max": 10,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": true,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 41,
    "successfull_dl": 62,
    "dable_attempts": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "vk.com"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 2
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 2
                            }
                        ]
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "pornxp.com"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            }
                        ]
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "x.com"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            }
                        ]
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "www.sugarinstant.com"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            }
                        ]
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "www.hotmovies.com"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 1
                            }
                        ]
                    }
                ]
            }
        ]
    },
    "dable_attempts_last": 1726047890909,
    "record_download_history": true,
    "history_limit_in_days": 60
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Click the icon to get into the VDH settings.  That gives you a short settings dialog.  Click the entry at the bottom that says `More settings`.  That opens a longer settings dialog.  Scroll it all the way to the bottom.  Click the button that says `Reload addon`.  Tell us what your VDH menu looks like after that.

**Comment by kammardekamm** [comment]
Thank you very much for your suggestion: the problem was solved!




 ____________________________

Il presente messaggio contiene informazioni da considerarsi strettamente riservate e confidenziali.

Ne è vietato l'uso improprio, la diffusione, la distribuzione o la riproduzione da parte di altre persone e/o entità diverse da quelle specificate.

Qualora lo abbiate ricevuto per errore, vi preghiamo di distruggere il messaggio, comunicando l'errata ricezione all’indirizzo mittente.

Rispetta l’ambiente. Non stampare questa mail se non strettamente necessario



This message contains information to be considered strictly private and confidential.

It is forbidden to misuse, disseminate, distribute, or reproduce by other persons and/or entities other than those specified.

If you received it in error, please destroy this message, communicating the mistaken receipt to the return address.

Respect the environment. Don’t print this email if it’s not strictly needed.

________________________________
Da: GrampaWildWilly ***@***.***>
Inviato: giovedì 12 settembre 2024 12:49
A: aclap-dev/video-downloadhelper ***@***.***>
Cc: kammardekamm ***@***.***>; Author ***@***.***>
Oggetto: Re: [aclap-dev/video-downloadhelper] Impossible to delete some files (Discussion #1248)


I do not work for VDH. I'm just another user.

Click the icon to get into the VDH settings. That gives you a short settings dialog. Click the entry at the bottom that says More settings. That opens a longer settings dialog. Scroll it all the way to the bottom. Click the button that says Reload addon. Tell us what your VDH menu looks like after that.
